server/src/test/scala/org/http4s/server/middleware/ChunkAggregatorSpec.scala

```diff
@@ -0,0 +1,52 @@
+package org.http4s.server.middleware
+
+import fs2._
+import org.http4s._
+import org.http4s.dsl._
+import org.specs2.matcher.MatchResult
+
+class ChunkAggregatorSpec extends Http4sSpec {
+
+  "ChunkAggregator" should {
+    "handle an empty body" in {
+      checkResponse(EmptyBody) { response =>
+        response.body.runLog.unsafeValue() must_=== Some(Vector.empty)
+      }
+    }
+
+    "handle a Pass" in {
+      val service: HttpService = HttpService.lift(_ => Pass.now)
+      ChunkAggregator(service).run(Request()).unsafeValue() must beSome(Pass)
+    }
+
+    "handle a single chunk" in {
```

Comment:
Would prefer this to be a property test. As all of the methods exposed are easy to test, and this is a fairly minimal spec.